src/pages/api/auth-connect/send-identity-validation-code.ts

```diff
@@ -0,0 +1,93 @@
+import { HttpStatus } from "@fwl/web";
+import { Handler } from "next-iron-session";
+
+import { IdentityStatus } from "@lib/@types/Identity";
+import { sendIdentityValidationCode } from "@lib/commands/sendIdentityValidationCode";
+import { ExtendedRequest } from "@src/@types/ExtendedRequest";
+import { insertTemporaryIdentity } from "@src/command/insertTemporaryIdentity";
+import { config, oauth2Client } from "@src/config";
+import { GraphqlErrors, MongoNoDataReturned } from "@src/errors";
+import { withAPIPageLogger } from "@src/middleware/withAPIPageLogger";
+import { withMongoDB } from "@src/middleware/withMongoDB";
+import withSession from "@src/middleware/withSession";
+import { getUser } from "@src/utils/getUser";
+import Sentry, { addRequestScopeToSentry } from "@src/utils/sentry";
+
+const handler: Handler = async (request: ExtendedRequest, response) => {
+  addRequestScopeToSentry(request);
+
+  try {
+    if (request.method === "POST") {
+      const { callbackUrl, identityInput } = request.body;
+
+      const user = await getUser(request.headers["cookie"] as string);
+
+      if (user) {
+        const decoded = await oauth2Client.verifyJWT<{ sub: string }>(
+          user.accessToken,
+          config.connectJwtAlgorithm,
+        );
+
+        const identity = {
+          status: IdentityStatus.VALIDATED,
```

Comment:
Why is it validated before sending the code? 🤔

Reply:
I hard coded the validation because, if the identity isn't verified with the verification flow, the add identity mutation isn't trigger. Let me know if it does make sense.